src/pages/index.tsx

```diff
@@ -19,26 +20,85 @@ import QuranInYearSection from '@/components/HomePage/QuranInYearSection';
 import ReadingSection from '@/components/HomePage/ReadingSection';
 import NextSeoWrapper from '@/components/NextSeoWrapper';
 import { Course } from '@/types/auth/Course';
+import Language from '@/types/Language';
+import { QuranFont } from '@/types/QuranReader';
+import { getDefaultWordFields, getMushafId } from '@/utils/api';
+import { makeVersesUrl } from '@/utils/apiPaths';
 import { fetchCoursesWithLanguages } from '@/utils/auth/api';
 import { isLoggedIn } from '@/utils/auth/login';
 import { getLanguageAlternates } from '@/utils/locale';
 import { getCanonicalUrl } from '@/utils/navigation';
 import getCurrentDayAyah from '@/utils/quranInYearCalendar';
 import { isMobile } from '@/utils/responsive';
 import withSsrRedux from '@/utils/withSsrRedux';
-import { ChaptersResponse } from 'types/ApiResponses';
+import { GetSsrPropsWithReduxContext } from '@/utils/withSsrRedux.types';
+import { ChaptersResponse, VersesResponse } from 'types/ApiResponses';
 import ChaptersData from 'types/ChaptersData';
 
+// Helper function to fetch Quran in Year verse for today
+async function fetchQuranInYearVerse(
```

Comment:
I don't believe this belongs here, we should move it to fetcher file where all API calls happen.